### ISSUE TITLE
Typos and words in the_shark_handbook.pdf

### ISSUE THREAD

**Issue by jkleiser** [shogundevel/shark#6]
Hi. I just downloaded and started reading the_shark_handbook.pdf to see how I can make use of shark. So far it looks interesting and well documented.
There are a few words that I think could be corrected or improved:
- You write that integers are "without a period" and that "floating-point numbers can have a period". I'd suggest you write "decimal point" instead of "period", as I think a period is what you find at the end of a sentence. ;-)
- Instead of "Characters and strings are the way shark haves to represent values ..." I would maybe have written "Characters and strings are the types shark uses to represent values ..." (or "haves" could have been replaced by "has").
- In "The following example searches trough a list" replace "trough" with "through".

I am curious to see if your shark bites, and if so, who's getting bitten and how.

**Comment by shogundevel** [comment]
Try finding as much typos as you can while reading it, im updating the
handbook as soon as wake up tomorrow and have my coffein dose. Thanks in
advance.

**Comment by jkleiser** [comment]
Could you first tell me how easy it would be to get shark running on a Mac (Intel, macOS 10.14.6), in the Terminal or in the browser?

**Comment by jkleiser** [comment]
In this pdf I see that the opening and closing quotes are different, like in this example:
```
x = “Hello.”
```
Here's what it looks like: 
![image](https://user-images.githubusercontent.com/107986/227735412-c70f02d4-d0e0-4df7-94bf-0f740e9b9a29.png)
When I look at the txt file in my BBEdit, it's lots of things indicating that BBEdit doesn't recognise which kind of coding is used. However, the quotes I see in the pdf may have been generated by some clever txt-to-pdf process.

**Comment by shogundevel** [comment]
You just need a standard C compiler, GCC or CLang, the whole C VM can be
built with a single command, look at build-shark-linux.sh for reference.
You can also try compiling the Java edition if you think the C version is
hard.

**Comment by shogundevel** [comment]
I dont know what youre doing with shark but, Do you liked using it so far?
If not, what would improve your experience? Thanks for your time in advance.

**Comment by jkleiser** [comment]
I'm sorry, but I have not installed shark yet. ;-)

**Comment by shogundevel** [comment]
The easiest way to test your install without writting any code is:
cd to the root of the install
run 'bin/shark tool build sharktool.shk sharktool.shar' to build a non
trivial shark program
run 'bin/shark sharktool.shar compile' to test the build success, you
should see a message in the console that is not an error
If at any point it doesnt work or an error happens, we have an issue.

**Comment by shogundevel** [comment]
I forgot, you seem to be reading the guide in .txt format, which probably
has an obscure windows encoding due to it being generated by microsoft
office. Switch to the PDF version for better reading ecperience.
Cheers.

**Comment by jkleiser** [comment]
Yes, these two lines
```
bin/shark tool build sharktool.shk sharktool.shar
bin/shark sharktool.shar compile
```
seemed to work as expected.
And when I did reopen the_shark_handbook.txt in my BBEdit, using the Windows Latin 1 encoding, I got a readable text. In line 128 which now looks like
```
x = “Hello.”
```
I can see that the two quote characters are different. They should both bee changed into "Hello.". Otherways copying code from this file (or the pdf) will cause problems.
